index.js

```diff
@@ -244,6 +244,9 @@ client.on("messageCreate", async (message) => {
     message.channel.send(
       "https://cdn.discordapp.com/attachments/664187293451419678/1034078773928267876/image0.jpg"
     );
+  if (message.content == "sexo?")

```

Comment:
it's okay, but I would rather prefer a switch, implement it that way and I'll merge